react/features/authentication/components/web/LoginQuestionDialog.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { WithTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+
+import { translate } from '../../../base/i18n/functions';
+import Dialog from '../../../base/ui/components/web/Dialog';
+
+/**
+ * The type of {@link LoginQuestionDialog}'s React {@code Component} props.
+ */
+interface IProps extends WithTranslation {
+
+    /**
+     * The handler.
+     */
+    handler: () => void;
+}
+
+/**
+ * Implements the dialog that warns the user that the login will leave the conference.
+ *
+ * @param {Object} props - The props of the component.
+ * @returns {React$Element}.
+ */
+function LoginQuestionDialog({ handler, t }: IProps) {
+    return (
+        <Dialog
+            ok = {{ translationKey: 'dialog.Yes' }}
+            onSubmit = { handler }
+            titleKey = { t('dialog.login') }>
+            <div>
+                { t('dialog.loginQuestion') }
+            </div>
+        </Dialog>
+    );
+}
+
+export default translate(connect()(LoginQuestionDialog));
```

Comment:
Do you need connect here?

Reply:
Also maybe you can use `useTranslation` hook and this way you won't need translate too and the props type will be simpler because you won't need withTranslation. WDYT?